charts/pega/values.yaml

```diff
@@ -87,6 +87,9 @@ global:
       url: "YOUR_DOCKER_REGISTRY"
       username: "YOUR_DOCKER_REGISTRY_USERNAME"
       password: "YOUR_DOCKER_REGISTRY_PASSWORD"
+    # To avoid exposing docker registry credentials, create secrets for docker registry credentials.
+    # Specify secret names as an array of comma-separated strings using the imagePullSecretNames parameter. e.g. ["secret1", "secret2"]
```

Comment:
Specify secret names as an array of comma-separated strings in double quotation marks using the imagePullSecretNames parameter. For example: ["secret1", "secret2"]